packages/app/src/types/pages.ts

```diff
@@ -1,10 +1,14 @@
-export enum Pages {
+export enum PoolPages {
```

Comment:
Why split into two enums here? I think is good to keep all routes on the app in a single enum.

Reply:
because when we have subroutes we need to don't have `/` upfront the path, so when I import `Pages.addLiquidity` in fact I'm don't importing a general page, but a PoolPage, this can bring problems mainly if we try to use it directly as `to` on Link, like I needed to do in some place of this changes:

```
<Link to={`../${PoolsPages.addLiquidity}` />
```

This route need not do be absolute, but relative.